packages/extension-api/src/extension-api.d.ts

```diff
@@ -4549,6 +4549,20 @@ declare module '@podman-desktop/api' {
      * Navigate to the Edit Provider Container Connection page
      */
     export function navigateToEditProviderContainerConnection(connection: ProviderContainerConnection): Promise<void>;
+
+    /**
+     * Allow to define custom route
+     * @param routeId
+     * @param commandId
+     */
+    export function register(routeId: string, commandId: string): Disposable;
+
+    /**
+     * Allow extension to navigate to a custom route
+     * @param routeId
+     * @param args
+     */
+    export function navigate(routeId: string, ...args: unknown[]): Promise<void>;
```

Comment:
still wondering if we need simple type (routeId, commandId) or a complex type ({id: command})

and the same for the navigate method

{
id: 
args?:
}

Reply:
I would keep it like for consistency with the others functions in the navigation namespace